src/style.css

```diff
@@ -1,3 +1,167 @@
+html {
+  .roboto-thin {
+    font-family: Roboto, sans-serif;
+    font-weight: 100;
+    font-style: normal;
+  }
+
+  .roboto-light {
+    font-family: Roboto, sans-serif;
+    font-weight: 300;
+    font-style: normal;
+  }
+
+  .roboto-regular {
+    font-family: Roboto, sans-serif;
+    font-weight: 400;
+    font-style: normal;
+  }
+
+  .roboto-medium {
+    font-family: Roboto, sans-serif;
+    font-weight: 500;
+    font-style: normal;
+  }
+
+  .roboto-bold {
+    font-family: Roboto, sans-serif;
+    font-weight: 700;
+    font-style: normal;
+  }
+
+  .roboto-black {
+    font-family: Roboto, sans-serif;
+    font-weight: 900;
+    font-style: normal;
+  }
+
+  .roboto-thin-italic {
+    font-family: Roboto, sans-serif;
+    font-weight: 100;
+    font-style: italic;
+  }
```

Comment:
why do you need this styles?
Just set font-family to `html` once

And why do you import all of this fonts to `fonts` folder if you import this font at html?